pysat/_instrument.py

```diff
@@ -273,6 +273,15 @@ def __init__(self, platform=None, name=None, tag='', inst_id='',
                 raise ValueError(' '.join(('Inputs platform and name must both',
                                            'be strings, or both None.')))
         else:
+            # Check if user supplied platform or name
+            if isinstance(platform, str) or isinstance(name, str):
+                warnings.warn(" ".join(("inst_module supplied along with",
```

Comment:
I had a question on whether warnings should be used or the logger. I found some guidance on the Python site:

```
warnings.warn() in library code if the issue is avoidable and the client application should be modified to eliminate the warning

logging.warning() if there is nothing the client application can do about the situation, but the event should still be noted
```

Good to know.